src/components/accountWizard/accountWizardStepOne.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import CopyField from '../copyField/copyField';
+
+const AccountWizardStepOne = ({ policySetup }) => (
+  <ul>
+    <li>
+      Create a new policy in the AWS{' '}
+      <a href="https://console.aws.amazon.com/iam" target="_blank" rel="noopener noreferrer">
+        Identity and Access Management
+      </a>
+    </li>
+    <li>
+      <p>In the JSON editor replace the contents with:</p>
+      <CopyField multiline value={policySetup} />
+    </li>
+    <li>
+      Click <strong>Review policy</strong>.
+    </li>
+    <li>
+      Name the policy <strong>Red-Hat-Cloud-Meter-policy</strong> and click <strong>Create policy</strong>.
+    </li>
+  </ul>
+);
+
+AccountWizardStepOne.propTypes = {
+  policySetup: PropTypes.string.isRequired
+};
+
+export { AccountWizardStepOne as default, AccountWizardStepOne };
```

Comment:
I'd suggest naming the component for what it is doing rather than the order in the wizard. Gets confusing if steps are added or steps are optional.